### ISSUE TITLE
Twice similar jobs with eck operator

### ISSUE THREAD

**Issue by domcar** [vvanholl/elasticsearch-prometheus-exporter#335]
Hi everyone,

i recently installed a ES Cluster using the eck operator and then install the prometheus exporter to get metrics. Everything works without problems, only I get the same result twice:
For example, if I query prometheus with: `es_os_mem_used_bytes`
I get :
```
es_os_mem_used_bytes{cluster="elastic-jaeger", container="elasticsearch", endpoint="https", instance="10.43.11.105:9200", job="elastic-jaeger-es-http", namespace="eck-cluster", node="elastic-jaeger-es-masters-0", nodeid="RCaOVA6vRCOrAwWN7wJaBg", pod="elastic-jaeger-es-masters-0", service="elastic-jaeger-es-http"}

es_os_mem_used_bytes{cluster="elastic-jaeger", container="elasticsearch", endpoint="https", instance="10.43.11.105:9200", job="elastic-jaeger-es-masters", namespace="eck-cluster", node="elastic-jaeger-es-masters-0", nodeid="RCaOVA6vRCOrAwWN7wJaBg", pod="elastic-jaeger-es-masters-0", service="elastic-jaeger-es-masters"
```

where the only difference is the `job name`.
The names seems to be the same as the services exposed by the cluster.

Anyone know what I did wrong or point me to the right direction?

**Comment by mindw** [comment]
@domcar The above suggests you have two services for the pod and both are scraped for metrics.

**Comment by domcar** [comment]
Yes, I have two services, but they were deployed automatically with the elastic search operator, not sure if I can simply remove one

**Comment by mindw** [comment]
just one of the scraping configs needs to be removed. How does the prometheus scraping of the service configured?

**Comment by domcar** [comment]
I have a serviceMonitor with a labelsSelector, but unfortunately the two services coming from the eck-operator have the same labels :( So I can change them manually and it's fine, but I can't find how I can modify them using the operator

**Comment by domcar** [comment]
Maybe this is more an issue on the operator side and not here, unless you used it too and know how to handle this
